File: src/main/java/com/ncc/aif/AidaDomainOntologiesCommon.java
package com.ncc.aif;

import org.apache.jena.rdf.model.Resource;
import org.apache.jena.rdf.model.ResourceFactory;

public final class AidaDomainOntologiesCommon {
    public static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/AidaDomainOntologiesCommon#";

    public static final Resource canHaveName = ResourceFactory.createResource(NAMESPACE + "CanHaveName");
    public static final Resource canHaveTextValue = ResourceFactory.createResource(NAMESPACE + "CanHaveTextValue");
    public static final Resource canHaveNumericValue = ResourceFactory.createResource(NAMESPACE + "CanHaveNumericValue");
}

File: src/main/java/edu/isi/gaia/AidaDomainOntologiesCommon.kt
package edu.isi.gaia

import org.apache.jena.rdf.model.ResourceFactory

object AidaDomainOntologiesCommon {
    val NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/AidaDomainOntologiesCommon#"

    val CanHaveName = ResourceFactory.createResource(NAMESPACE + "CanHaveName")!!
    val CanHaveTextValue = ResourceFactory.createResource(NAMESPACE + "CanHaveTextValue")!!
    val CanHaveNumericValue = ResourceFactory.createResource(NAMESPACE + "CanHaveNumericValue")!!
}
